esx_service/utils/vmdk_utils.py

```diff
@@ -163,11 +163,14 @@ def get_volumes(tenant_re):
                     logging.debug("get_volumes: cannot find tenant_name for tenant_uuid=%s", sub_dir_name)
                     logging.debug("get_volumes: path=%s root=%s sub_dir_name=%s",
                                 path, root, sub_dir_name)
-                    for file_name in list_vmdks(root):
-                        volumes.append({'path': root,
-                                        'filename': file_name,
-                                        'datastore': datastore,
-                                        'tenant' : auth_data_const.ORPHAN_TENANT})
+
```

Comment:
Line 130 looks broken to me - `path` is .../dockvol, and should be ../dockvol/__DEFAULT.